index.html

```diff
@@ -334,8 +334,8 @@ <h2>New Projects and Exploration</h2>
 		</div>
 
 		<div style="position:absolute;top:0px;width:100vw;height:100vh;" class="carousel-controls">
-			<a style="position:absolute;top:0px;left:0px;color:black" href="#" class="animate-button--previous">Previous</a>
-			<a style="position:absolute;top:0px;right:0px;color:black" href="#" class="animate-button--next">Next</a>
+			<a style="position:absolute;top:45%;left:2%;color:black" href="#" class="animate-button--previous"><img src="media/images/general/left-arrow.svg" class="slide-change"></a>
```

Comment:
%'s work fine for the arrows since they're in a container that is the size of the screen. There might be some instances in future code on the text where you'll want to use vw and vh units instead (just a heads up for when you're working with text).

Reply:
ah, right. I'll keep that in mind for the future.